pkg/sql/schemachanger/scjob/job.go

```diff
@@ -162,6 +180,36 @@ func makeTargetStates(
 	return ts
 }
 
+func getStmtsForTestingMetadata(j *jobs.Job) []string {
+	// This is a hack to get individual statements back out of the concatenated
+	// statements.
+	return strings.Split(j.Payload().Statement, "; ")
+}
+
+func clearDescriptorJobIDs(
+	ctx context.Context,
+	txn *kv.Txn,
+	descriptors *descs.Collection,
+	descIDs []descpb.ID,
+	jobID jobspb.JobID,
```

Comment:
This is almost the same function as `setDescriptorJobIDs`, perhaps they could both be unified as `CompareAndSetDescriptorJobIDs(..., expectedJobID jobspb.JobID, newJobID jobspb.JobID)`.